.github/workflows/nmodl-ci.yml

```diff
@@ -26,7 +26,7 @@ jobs:
     name: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ ubuntu-18.04, macos-10.15, ubuntu-20.04, macos-11 ]
+        os: [ ubuntu-18.04, macos-10.14, ubuntu-20.04, macos-11 ]
```

Comment:
Is this documented anywhere? It doesn't seem to be listed here: https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources

Reply:
May be worth verifying : **macos-10.14 image is using macos-10.15 #2347** : https://github.com/actions/virtual-environments/issues/2347

If there is no option to use 10.14, we can let it go!